qdrant/v0.11.x/integrations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Integrations
+weight: 52
+---
+Qdrant is a vector database, performing the 
```

Comment:
typo?

Reply:
Sorry about that, I had the intro already written, but somehow not committed. It's done.